test/suites/beta/expiration/suite_test.go

```diff
@@ -80,6 +80,28 @@ var _ = BeforeEach(func() {
 					NodeClassRef: &corev1beta1.NodeClassReference{
 						Name: nodeClass.Name,
 					},
+					Requirements: []v1.NodeSelectorRequirement{
+						{
+							Key:      v1.LabelOSStable,
+							Operator: v1.NodeSelectorOpIn,
+							Values:   []string{string(v1.Linux)},
+						},
+						{
+							Key:      corev1beta1.CapacityTypeLabelKey,
+							Operator: v1.NodeSelectorOpIn,
+							Values:   []string{corev1beta1.CapacityTypeOnDemand},
+						},
+						{
+							Key:      v1beta1.LabelInstanceCategory,
+							Operator: v1.NodeSelectorOpIn,
+							Values:   []string{"c", "m", "r"},
+						},
+						{
+							Key:      v1beta1.LabelInstanceGeneration,
+							Operator: v1.NodeSelectorOpGt,
+							Values:   []string{"2"},
+						},
+					},
```

Comment:
Does it make sense for us to just have a test.DefaultRequirements() that we add to every NodePool, and if we want it to be something else, we just set a nil []v1.NodeSelectorRequirement? I can imagine adding in a new suite, or a new test and forgetting these, requiring suite test writers to be aware of these provisioning suggestions.